### ISSUE TITLE
Doesn't seem to work 

### ISSUE THREAD

**Issue by Volko61** [adrienbrault/hf-gguf-to-ollama#3]
`PS C:\Users\Volko> dagger -m github.com/adrienbrault/hf-gguf-to-ollama  --url Volko76/Qwen2.5-0.5B-Instruct-French-LoRa-GGUF --quant Q4_K_M call modelfile
✔ connect 0.4s
✘ loading module 4.6s
! failed to serve module: input: moduleSource.asModule resolve: failed to create module: select: failed to update codegen and runtime: failed to generate code: failed to call sdk module codegen: select: call function "Codegen": process "/runtime" did not complete successfully: exit code: 2
  ✔ finding module configuration 1.8s
  ✘ serving module 2.8s
  ! input: moduleSource.asModule resolve: failed to create module: select: failed to update codegen and runtime: failed to generate code: failed to call sdk module codegen: select: call function "Codegen": process "/runtime" did not complete successfully: exit code: 2
    ✘ ModuleSource.asModule: Module! 2.8s
    ! failed to create module: select: failed to update codegen and runtime: failed to generate code: failed to call sdk module codegen: select: call function "Codegen": process "/runtime" did not complete successfully: exit code: 2

Setup tracing at https://dagger.cloud/traces/setup. To hide: export STOPIT=1

Error: failed to serve module: input: moduleSource.asModule resolve: failed to create module: select: failed to update codegen and runtime: failed to generate code: failed to call sdk module codegen: select: call function "Codegen": process "/runtime" did not complete successfully: exit code: 2

Stdout:
invoke: failed to create codegen base: failed to add template: could not list module source entries: input: container.from.withoutEntrypoint.withMountedCache.withMountedDirectory.withDirectory.withDirectory.withWorkdir.withExec.withExec.withExec.withDirectory.directory resolve: process "bun install --no-verify --no-progress --summary ./sdk" did not complete successfully: exit code: 1

Stderr:
bun add v1.1.26 (0a37423b)
Saved lockfile

$ cp ./test/sdk.index-fixed.ts ./sdk/index.ts
cp: cannot stat './test/sdk.index-fixed.ts': No such file or directory
error: postinstall script from "" exited with 1
PS C:\Users\Volko>
`

**Comment by richtong** [comment]
Yes I got exactly the same error. Too bad, looked really promising all this dagger stuff :-(

The error mesages pretty hard to understand
